app/Makefile

```diff
@@ -54,35 +52,36 @@ endif # } PDIR
 APPDIR = .
 LDDIR = ../ld
 
-TARGET_LDFLAGS = 				\
+TARGET_LDFLAGS = 			\
 	-nostdlib 				\
 	-Wl,-EL 				\
-	--longcalls 				\
+	--longcalls 			\
 	--text-section-literals
 
 LD_FILE = $(LDDIR)/nodemcu.ld
 
-COMPONENTS_eagle.app.v6 = 			\
+COMPONENTS_eagle.app.v6 = 		\
 	user/libuser.a 				\
 	crypto/libcrypto.a			\
 	driver/libdriver.a 			\
-	platform/libplatform.a 			\
+	platform/libplatform.a		\
 	task/libtask.a 				\
 	libc/liblibc.a 				\
 	lua/liblua.a 				\
 	lwip/liblwip.a 				\
 	smart/smart.a 				\
 	spiffs/spiffs.a 			\
 	fatfs/libfatfs.a			\
-	pm/libpm.a				\
-	esp-gdbstub/libgdbstub.a		\
-	net/libnodemcu_net.a			\
-	mbedtls/libmbedtls.a      		\
-	modules/libmodules.a			\
+	pm/libpm.a					\
+	esp-gdbstub/libgdbstub.a	\
+	net/libnodemcu_net.a		\
+	mbedtls/libmbedtls.a    	\
+	modules/libmodules.a		\
 	smart/smart.a 				\
 	uzlib/libuzlib.a 			\
 	$(OPT_SEL_COMPONENTS)
-
+# Ensure that the Espresif SDK is searched before the tool-chain's SDK (if any)
```

Comment:
I don't think this comment is accurate here. Copy paste slip?

Reply:
I will check and remove